packages/pdf-viewer/pdfSource.test.ts

```diff
@@ -0,0 +1,41 @@
+import { PdfData } from './pdfSource';
+import * as pdfjsLib from 'pdfjs-dist';
+
+const path = require('path');
+const fs = require('fs');
+
+pdfjsLib.GlobalWorkerOptions.workerSrc = require('pdfjs-dist/legacy/build/pdf.worker.entry');
+
+const pdfFilePath = path.resolve('config/welcome.pdf');
+
+describe('pdfData', () => {
+	const file = new Uint8Array(fs.readFileSync(pdfFilePath));
```

Comment:
Only use async methods in tests. Sync methods prevent other tests from running concurrently.